Resideo_eshopping/resideo_eshopping/lib/Screens/productdetail.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:resideo_eshopping/model/eshopping_model.dart';
+import 'package:resideo_eshopping/Screens/addUserDetails.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class StarDisplay extends StatelessWidget {
```

Comment:
Change file name to maintain naming conventions (lower case separated by under score)
ex: product_detail.dart